aidial_adapter_openai/app.py

```diff
@@ -58,6 +56,16 @@ async def handle_exceptions(call):
         )
 
 
+def get_api_version(request: Request):
+    api_version = request.query_params.get("api-version", "")
+
+    return (
```

Comment:
You could just use `get(key[, default])` with default value like this:
```
return api_versions_mapping.get(api_version, api_version)
```

Reply:
fixed, thank you